etc/test/passes.cpp

```diff
@@ -97,7 +97,16 @@ SOURCE_TEST( analyze, ConsistencyCheckPass, source, true, , );
 SOURCE_TEST( transform, AstDumpSourcePass, source, true, , );
 SOURCE_TEST( transform, AstDumpDotPass, source, true, , );
 
-// NumericExecutionPass
+SOURCE_TEST( execute, NumericExecutionPass, source, true, , );
```

Comment:
The problem happens in this test case.

Trace-output (just added some couts):
```
StateInitializationVisitor::Specification
StateInitializationVisitor::InitDefinition
StateInitializationVisitor::FunctionDefinition
ExecutionVisitor::ReferenceLiteral test_skip
ExecutionVisitor::UpdateRule [v] @test_skip
StateInitializationVisitor::Specification updates fired!
AgentScheduler::step
AgentScheduler::collectAgents
ExecutionVisitor::collectAgents::loop [Agent] @c0 = Agent Agent -> [v] v undef
libcasm-fe-check: casm-fe/src/execute/NumericExecutionPass.cpp:1940: std::vector<Agent> AgentScheduler::collectAgents() const: Assertion `IR::isa< ReferenceConstant >( value )' failed.
```
Somewhere `[v] @test_skip` becomes `[v] v undef`. But this also happens in master, so this test case should be moved to another PR to fix the problem there.

Reply:
@emmanuel099 :+1: in this case I will open a bug with this issue and revert this change in the `etc/test/passes.cpp` unit test case file